rules/do-you-know-the-quickest-way-to-get-your-windows-soe-up-and-running/rule.md

```diff
@@ -9,87 +9,82 @@ authors:
 
-1. Logon and configure Windows 10
-2. Install Windows Updates
-3. Join laptop to the Domain (or run a Logon-Script.bat to get printers, Word templates, etc. Contact SysAdmin to join laptop to the domain)
-4. Install and run a Chocolatey script to get most of the software
-5. Install all rest of the standard Software that does not support Chocolatey \[SSW Only]   
-6. In case you lose your laptop, it is a good idea to stick a business card to the bottom
-7. For branding reasons, stick your company sticker to the laptop 
-    
-![Figure: SSWLoginScript.bat](LoginScript.jpg)
+1. Logon and configure Windows
+2. (Optional) Join the Domain
+3. Install Windows Updates
+4. Install driver & firmware updates (tip: use an app from the manufacturer, e.g. [Intel DSA](https://www.intel.com.au/content/www/au/en/support/detect.html))
+5. Run a script to get printers, Word templates, etc.
+6. Install and run a Chocolatey script to get most of the software
+7. Install any software not supported by Chocolatey
+8. In case you lose your laptop, it is a good idea to stick a business card to the underneath
+9. For branding, stick your company sticker to the laptop - or even better, a custom skin - see [Do you brand your assets?](https://www.ssw.com.au/rules/brand-your-assets/)
 
-![Figure: SSWPackages.ps1 which runs Chocolatey command to install SOE software](ChocolateyScript.jpg)
+![Figure: Example login/branding script](login-script.png)
 
-![Figure: SOE Software that is not supported by Chocolatey](NonChocolateyApp.jpg)
+![Figure: Example package manager - using chocolatey to install required software](choco.png)
 
-![Figure: Company branding on laptop](SSWLaptopBranding.jpg)
+![Figure: Company branding on laptop](ssw-skin.jpeg)
 
 8. If you are preparing the machine for someone else or migrating an old PC, you could send an email based on the template below. Don't forget to change the names where necessary:
 
-::: email-template  
+::: email-template\
 |          |     |
 | -------- | --- |
-| To:      | Ana |
+| To:      | Bob Northwind|
 | Cc:      | YYY |
-| Subject: | Your new computer |  
+| Subject: | Your new computer |\
 ::: email-content  
 
-### Hey Ana, 
+### Hey Bob,
 
 Your new computer is ready. ⭐️  
 
-* Machine name is COBRA
-* Model: ThinkPad X1 Carbon Gen8
-* Operating System: Windows 10 Enterprise installed and activated
+* Machine name: Bob-SurfacePro9
+* Model: Surface Pro 9
+* Operating System: Windows 11 Enterprise  
 
- **Done** - I have followed an SSW Rule: [https://rules.ssw.com.au/do-you-know-the-quickest-way-to-get-your-windows-10-soe-up-and-running](/do-you-know-the-quickest-way-to-get-your-windows-10-soe-up-and-running) and this included: 
+I have followed an SSW Rule: [https://rules.ssw.com.au/do-you-know-the-quickest-way-to-get-your-windows-10-soe-up-and-running](/do-you-know-the-quickest-way-to-get-your-windows-10-soe-up-and-running) and this included: 
```

Comment:
Change URL to https://rules.ssw.com.au/do-you-know-the-quickest-way-to-get-your-windows-soe-up-and-running (removing the Windows 10 thing)

Reply:
Done